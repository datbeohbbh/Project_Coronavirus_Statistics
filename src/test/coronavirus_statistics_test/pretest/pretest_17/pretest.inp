23 24
q 1 16 339
q 1 21 88
? 4 2
q 1 3 378
q 3 16 79
? 4 2
q 3 16 98
q 2 21 29
q 2 21 31
? 4 2
q 1 3 476
q 3 16 83
q 3 3 167
? 4 1
q 1 17 492
q 2 17 217
q 2 21 19
? 7
q 2 3 422
? 6
? 6
? 7
? 4 3
? 7