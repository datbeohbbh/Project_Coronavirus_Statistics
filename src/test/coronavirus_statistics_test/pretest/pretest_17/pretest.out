Query ? 4 2 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 339
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 260
-> Has recovered : 0
-> Was dead : 79

Query ? 4 2 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 162
-> Has recovered : 0
-> Was dead : 177

Query ? 4 1 :
The answer is the city with the index 21 
- Information of the city 21 -
-> Number of current patients : 28
-> Has recovered : 60
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 16
Quantity: 260 people

Query ? 6 :
The city with the highest recovery is city with the index 3
Quantity: 422 people

Query ? 6 :
The city with the highest recovery is city with the index 3
Quantity: 422 people

Query ? 7 :
The city with the highest death is city with the index 16
Quantity: 260 people

Query ? 4 3 :
The answer is the city with the index 3 
- Information of the city 3 -
-> Number of current patients : 265
-> Has recovered : 422
-> Was dead : 167

Query ? 7 :
The city with the highest death is city with the index 16
Quantity: 260 people

